Pipelines/Templates/tests.yaml

```diff
@@ -78,50 +45,49 @@ steps:
-        Write-Output '             Begin PlayMode Tests Logs               '
-        Write-Output '====================================================='
-
-        Get-Content $logFile
-
-        Write-Output '====================================================='
-        Write-Output '              End PlayMode Tests Logs                '
-        Write-Output '====================================================='
-      }
-      else {
-        Write-Output 'PlayMode Tests log missing!'
       }
-
-      if (Test-Path $resultFile) {
+      finally {
         Write-Output '====================================================='
-        Write-Output '            Begin PlayMode Tests Results             '
-        Write-Output '====================================================='
-
-        Get-Content $resultFile
-
-        Write-Output '====================================================='
-        Write-Output '             End PlayMode Tests Results              '
+        Write-Output '               PlayMode Tests Finished               '
         Write-Output '====================================================='
+        if (Test-Path $logFile) {
+          Write-Output '====================================================='
+          Write-Output '             Begin PlayMode Tests Logs               '
+          Write-Output '====================================================='
+          Get-Content $logFile
+          Write-Output '====================================================='
+          Write-Output '              End PlayMode Tests Logs                '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests log missing!'
+        }
+        if (Test-Path $resultFile) {
+          Write-Output '====================================================='
+          Write-Output '            Begin PlayMode Tests Results             '
+          Write-Output '====================================================='
+          Get-Content $resultFile
+          Write-Output '====================================================='
+          Write-Output '             End PlayMode Tests Results              '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests results missing!'
+        }
+      Write-Output $($proc.ExitCode)
       }
-      else {
-        Write-Output 'PlayMode Tests results missing!'
-      }
-
-      Write-Output "Exit code: $($proc.ExitCode)"
-    }
-  displayName: 'Run PlayMode tests'
 
-- task: PublishTestResults@2
-  displayName: 'Publish Test Results'
-  inputs:
-    testResultsFormat: NUnit
-    testResultsFiles: 'test*.xml'
-    searchFolder: $(Agent.TempDirectory)
-    failTaskOnFailedTests: true
+    displayName: 'Run PlayMode tests'
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: 'D:\a\_work\1\a'
```

Comment:
If we're going to keep the copy, can this reference `$(Build.ArtifactStagingDirectory)` instead of what it might point to?

Reply:
Which also makes me slightly confused about what this step is doing. It's copying the entire contents of `$(Build.ArtifactStagingDirectory)` to `$(Build.ArtifactStagingDirectory)\Temp`? Or am I misreading this?

Reply:
You are right that task was not actually doing anything helpful